twinkle.css

```diff
@@ -42,3 +63,40 @@
 #twinkle-config-content input[type=checkbox] {
 	margin-right: 2px;
 }
+
+/* Twinkle config */
+#twinkle-config {
+	border: 1px solid #666;
+	background: var(--twinkle-bgcolor-dialog, #F0F8FF);
+	color: inherit;
+}
+
+#twinkle-config label {
+	color: var(--color-base, #202122);
+}
+
+#twinkle-config-titlebar {
+	background: var(--twinkle-bgcolor-titlebar, #BCCADF);
+	color: inherit;
+}
+
+#twinkle-config-buttonpane {
+	background: var(--twinkle-bgcolor-titlebar, #BCCADF);
+	color: inherit;
+	padding: 0.5em;
+}
+
+.twinkle-config-helptip {
+	font-size: 90%;
+	color: var(--color-subtle, #54595d);
+}
+
+@media screen {
+	html.skin-theme-clientpref-night #twinkle-config-headerbox {
+		background: #532B18;
+}
```

Comment:
A closing brace is missing here. Same issue in the other media query block as well.

Reply:
Fixed